src/BackgroundTasks/Task/Job.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace ILIAS\BackgroundTasks\Task;
+
+use ILIAS\BackgroundTasks\Observer;
+use ILIAS\BackgroundTasks\Value;
+use ILIAS\BackgroundTasks\Task;
+
+/**
+ * Interface Job
+ *
+ * @package ILIAS\BackgroundTasks\Task
+ *
+ *          A Task, which can be run without any interaction with the user, such as zipping files
+ *          or just collecting some data
```

Comment:
Nice explanation. Maybe move this to your glossary as well, the "without interaction" part is missing there.